packages/local-relay/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@fitbit/local-relay",
+  "version": "1.8.0-pre.0",
+  "description": "Local implementation of the Developer Relay",
+  "author": "Fitbit, Inc.",
+  "homepage": "https://github.com/Fitbit/developer-bridge/tree/main/packages/local-relay#readme",
+  "license": "BSD-3-Clause",
+  "main": "lib/local-relay.js",
+  "publishConfig": {
+    "registry": "http://registry.npmjs.org/"
+  },
+  "repository": "github:Fitbit/developer-bridge",
+  "scripts": {
+    "build": "rm -rf lib tsconfig.tsbuildinfo && tsc -b",
+    "prepublishOnly": "yarn run build"
+  },
+  "bugs": {
+    "url": "https://github.com/Fitbit/developer-bridge/issues"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.16.0",
+    "@babel/preset-env": "^7.16.4",
+    "@babel/preset-typescript": "^7.16.0",
+    "@types/express": "^4.17.13",
+    "@types/jest": "^27.0.2",
+    "@types/node": "^16.10.2",
+    "@types/supertest": "^2.0.11",
+    "@types/websocket": "^1.0.4",
+    "babel-jest": "^27.3.1",
+    "jest": "^27.3.1",
+    "supertest": "^6.1.6"
+  },
+  "dependencies": {
+    "express": "^4.17.1",
+    "typescript": "^4.4.3",
+    "uuid": "^8.3.2",
+    "websocket": "^1.0.34"
+  },
+  "bin": {
+    "fitbit": "./lib/cli.js"
+  },
+  "files": [
+    "/lib/!(*.test|*.spec).{js,d.ts}",
+    "/lib/!(testUtils)**/!(*.test|*.spec).{js,d.ts}",
+    "/lib/**/*.json"
+  ],
+  "engines": {
+    "node": ">=8.6.0"
```

Comment:
This can safely be >=12.0.0, last LTS.

Reply:
`https://github.com/Fitbit/developer-bridge/pull/182/commits/fa15c8c6999dae5376d7440d49487249ae424bc6`